controller/comments.controller.js

```diff
@@ -27,3 +27,12 @@ exports.getAllCommentsFromID = (req, res, next) => {
     })
     .catch(next);
 };
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  deleteComments(comment_id)
```

Comment:
Models should use the naming convention that utilises a different verb for the method it is associated with. Your controller here uses (correctly) delete - the method used - in its name. So the model could use "remove" instead and be called removeCommentById. We should use this convention for all controllers and models 👍